src/extension/background-script/actions/accounts/unlock.ts

```diff
@@ -31,7 +31,7 @@ const unlock = (message: MessageAccountUnlock) => {
   }
 
   // if everything is fine we keep the password in memory
-  state.setState({ password });
+  await chrome.storage.session.set({ password });
```

Comment:
I would not like have code that we know will not work in here I think. 
This will be double work. 
If you go that route then also check that we are able to go that way and how. 

(especially when it is done in all the places https://github.com/getAlby/lightning-browser-extension/pull/1124/files#diff-1af05f60841943485ce3642d0c9cb25cfd58f148f6caec4162513a08bf571eb6R10 - all those need to change again)

Reply:
Updated it. Tried to centralize the usage. Works. Not sure if this is how you use `state` but it's better than. Before. wdyt?  
Need to adapt the code for MV2 only in one place now.